src/clj/ataru/applications/automatic_payment_obligation.clj

```diff
@@ -0,0 +1,24 @@
+(ns ataru.applications.automatic-payment-obligation
+  (:require [ataru.background-job.job :as job]
+            [ataru.koodisto.koodisto-codes :as codes]
+            [ataru.applications.application-store :as application-store]))
+
+(defn nationality-finland? [{:keys [kansalaisuus]}]
+  (some true? (map #(= codes/finland-country-code %) kansalaisuus)))
+
+(defn automatic-payment-obligation-job-step
+  [{:keys [person-oid]}
+   {:keys []}]
+  (let [oids (->> (application-store/onr-applications person-oid)
+                  (filter nationality-finland?)
```

Comment:
Olettaisin, että tässä pitäis käyttää ONR:stä löytyvää kansalaisuutta ja ehkä vaan yksilöidyiltä henkilöiltä.